architecture/networking/network_plugins.adoc

```diff
@@ -50,30 +32,42 @@ Ansible-based installation procedure. See the
 xref:../../architecture/networking/sdn.adoc#architecture-additional-concepts-sdn[OpenShift SDN] section
 for more information.
 
+== Third-Party SDN plug-ins
```

Comment:
I think you can just drop the OpenShift SDN section here. From the Overview, you can make "Software defined network (SDN) plug-ins" a hyperlink to the existing OpenShift SDN topic. Also, for the sake of modular docs, we should probably also rename "Overview", or drop the section altogether, combining the content with the "OpenShift SDN" section.

Reply:
I thought it would be a good idea to highlight that the OpenShift SDN is the default, but it can probably be trimmed down to something like just the link to the section. Thanks for the comment! I'll drop the "Overview" title at the least, and make sure it's more about the topic.